src/lib/connectors/magicConnectConnector.ts

```diff
@@ -21,10 +22,14 @@ import { getAddress } from 'ethers/lib/utils.js'
 export interface MagicConnectorOptions {
   apiKey: string
   magicSdkConfiguration?: MagicSDKAdditionalConfiguration
+  networks?: EthNetworkConfiguration[]
```

Comment:
Added a new option `networks` to be as close as possible to the current types without breaking the API